packages/react-composites/tests/browser/call/HorizontalGallery.test.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { test } from './fixture';
+import { expect } from '@playwright/test';
+import { v1 as generateGUID } from 'uuid';
+
+test.describe('HorizontalGallery tests', async () => {
+  test.beforeEach(async ({ pages, users, serverUrl }) => {
+    // Each test *must* join a new call to prevent test flakiness.
+    // We hit a Calling SDK service 500 error if we do not.
+    // An issue has been filed with the calling team.
+    const newTestGuid = generateGUID();
+    const user = users[0];
+    user.groupId = newTestGuid;
+
+    // Load different locale for locale tests
+    const page = pages[0];
+    await page.goto(
+      buildCallingUrl(
+        serverUrl,
+        {
```

Comment:
There would be a setup step for this but I need to match this type to CallAdapterState. But I cannot import it in tests. Any ideas?